jujugui/static/gui/src/app/state/state.js

```diff
@@ -82,6 +95,76 @@ const State = class State {
   }
 
   /**
+    Check the supplied block of path to see if this is a user store path or
+    not. These options are becuase the store path can come in many forms and
+    we need to know if it's a user store path for both parsing and
+    generating the path.
+    @return {Boolean}
+  */
+  _isUserStorePath(block) {
+    const seriesList = this.seriesList;
+    // Store path ex) django/bundle/42
+    if (block.length === 3 && block[1] === 'bundle') {
```

Comment:
Why bundle is a special case? It should be a series like all the others and it is included in jujulib.URL.SERIES.